### PR TITLE
corrected the add support error

### PR DESCRIPTION
**Pull request by birey**

### CODE REVIEW

**Comment by chenxiaolong** [comment]
@chenxiaolong-bot test this please (This is a message for the build server)

File: data/devices/samsung/02_galaxy_a_j.yml [hunk 1]
```diff
@@ -76,3 +76,36 @@
     extra:
       - /dev/block/platform/13540000.dwmmc0/by-name/CDMA-RADIO
       - /dev/block/platform/13540000.dwmmc0/by-name/RADIO
+
+ name: Samsung Galaxy E5
```
Comment (chenxiaolong) [inline data/devices/samsung/02_galaxy_a_j.yml]: This line should be:

```
-  name: Samsung Galaxy E5`
```

The current syntax is invalid.

Comment (birey) [inline data/devices/samsung/02_galaxy_a_j.yml]: I have corrected the errror. Thankyou.

**Comment by chenxiaolong** [comment]
@chenxiaolong-bot test this please (This is a message for the build server)

**Pull request merged**